test_e2e.sh

```diff
@@ -13,5 +13,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-./test/e2e/ci.sh
+CHECK_DOCS_ONLY=$("test/check_docs_only.sh")
+if [ -z "$CHECK_DOCS_ONLY" ]; then
+    ./test/e2e/ci.sh
+    exit 0
+fi
+echo "WARNING: The tests were skipped because only changes on the docs were faced"
```

Comment:
I think it is missing else here otherwise will output always. Right?

Reply:
Could you create 2 draft PRs against Kubebuilder for us to check if it is working ?
1 Pr should have a change that will trigger the tests and in the other, a change that will not trigger the tests.